platform/podjs/src/browserPod.ts

```diff
@@ -97,26 +136,52 @@ class FileUrl {
         );
     }
 
+    /**
```

Comment:
Same as above, not shown in the documentation unless the class itself has a doc string.